app/controllers/api/users_controller.rb

```diff
@@ -26,5 +26,21 @@ def login
         }
       }.to_json
     end
+
+    def show
+      user = User.find_by(id: params[:id])
+      if user.nil?
```

Comment:
Let's add a new blank line between "querying data" section and a conditional to make the code cleaner.